server/channels/app/upload.go

```diff
@@ -230,7 +230,9 @@ func (a *App) UploadData(c request.CTX, us *model.UploadSession, rd io.Reader) (
 			return nil, err
 		}
 		if written < minFirstPartSize && written != us.FileSize {
-			a.RemoveFile(uploadPath)
+			if err = a.RemoveFile(uploadPath); err != nil {
+				c.Logger().Warn("Failed to remove file", mlog.Err(err))
+			}
```

Comment:
Let's use a different variable name to not overwrite the error that gets checked in L237.